server/request_test.go

```diff
@@ -17,6 +17,8 @@ type mockedSigner struct {
 	signatureToReturn identity.Signature
 }
 
+var testMysteriumApiUrl = "http://testUrl"
```

Comment:
We shouldn't depend that url is `mysterium` - it can be just `testRequestURL`.
Also, `URL` has to be upper-case.